mindocr/postprocess/det_db_postprocess.py

```diff
@@ -68,7 +68,10 @@ def _postprocess(self, pred: Union[Tensor, Tuple[Tensor], np.ndarray], **kwargs)
             pred = pred[self._names[self._name]]
         if isinstance(pred, Tensor):
             pred = pred.asnumpy()
-        pred = pred.squeeze(1)
+        if len(pred.shape) == 4 and pred.shape[1] != 1:  # pred shape (N, 3, H, W)
+            pred = pred[:, :1, :, :]  # only need the first output
```

Comment:
DBNet predicts a single channel heatmap. Why would it have channel shape 3?

Reply:
paddle-ocr models may have (N, 3, H, W) output shape and currently use the same postprocess transform